src/server/components/dashboard/listsItems/itemsRouter.ts

```diff
@@ -38,18 +50,31 @@ listRouter.param("listId", async (req, res, next, listId) => {
   }
 });
 
-listRouter.all("/:listId", listsEditController);
-listRouter.all("/:listId/*", redirectIfUnauthorised);
```

Comment:
redirectIfUnauthorised didn't actually do any redirecting.

I've refactored to `validateAccessToList`

Reply:
it probably should (to the login page, or the last "accessible" page with a notification error). at the moment the user can't recover from the error apart from hitting the back button